Segfault when trying to document Redshift database

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x41f7cf6]

goroutine 1 [running]:
database/sql.(*Rows).close(0x0, 0x0, 0x0, 0x0, 0x0)
        /Users/k1low/.anyenv/envs/goenv/versions/1.12.5/src/database/sql/sql.go:2976 +0x66
database/sql.(*Rows).Close(0x0, 0x48c2a0c, 0x172)
        /Users/k1low/.anyenv/envs/goenv/versions/1.12.5/src/database/sql/sql.go:2972 +0x33
github.com/k1LoW/tbls/drivers/postgres.(*Postgres).Analyze(0xc0003397c0, 0xc0000dd4f0, 0x498d880, 0xc0003b6f40)
        /Users/k1low/src/github.com/k1LoW/tbls/drivers/postgres/postgres.go:131 +0x2f8a
github.com/k1LoW/tbls/datasource.Analyze(0x7ffeefbfe421, 0xa0, 0x0, 0x0, 0x0)
        /Users/k1low/src/github.com/k1LoW/tbls/datasource/datasource.go:74 +0x683
github.com/k1LoW/tbls/cmd.glob..func2(0x4e5dbc0, 0xc0001f7460, 0x1, 0x1)
        /Users/k1low/src/github.com/k1LoW/tbls/cmd/doc.go:72 +0x111
github.com/spf13/cobra.(*Command).execute(0x4e5dbc0, 0xc0001f7430, 0x1, 0x1, 0x4e5dbc0, 0xc0001f7430)
        /Users/k1low/go/pkg/mod/github.com/spf13/cobra@v0.0.4/command.go:766 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0x4e5d4a0, 0x0, 0x0, 0x45eabea)
        /Users/k1low/go/pkg/mod/github.com/spf13/cobra@v0.0.4/command.go:850 +0x2fc
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/k1low/go/pkg/mod/github.com/spf13/cobra@v0.0.4/command.go:800
github.com/k1LoW/tbls/cmd.Execute()
        /Users/k1low/src/github.com/k1LoW/tbls/cmd/root.go:59 +0x2e
main.main()
        /Users/k1low/src/github.com/k1LoW/tbls/main.go:32 +0x20
```

```bash
$ tbls version
1.15.1
```

```bash
$ go version
go version go1.12.5 darwin/amd64
```

Please let me know if there's any other information I can provide to help debugging!

--- a/drivers/postgres/postgres.go
+++ b/drivers/postgres/postgres.go
@@ -308,10 +308,10 @@ ORDER BY x.indexrelid
 				Def:   indexDef,
 				Table: &table.Name,
 			}
-			idxs := indkeyToInts(indkey)
-			for _, idx := range idxs {
-				index.Columns = append(index.Columns, table.Columns[idx-1].Name)
-			}
+			// idxs := indkeyToInts(indkey)
+			// for _, idx := range idxs {
+			// 	index.Columns = append(index.Columns, table.Columns[idx-1].Name)
+			// }
 
 			indexes = append(indexes, index)
 		}
@@ -330,12 +330,12 @@ ORDER BY x.indexrelid
 				return err
 			}
 			l.constraint.Table = &table.Name
-			if l.conkey != "" {
-				idxs := colkeyToInts(l.conkey)
-				for _, idx := range idxs {
-					l.constraint.Columns = append(l.constraint.Columns, table.Columns[idx-1].Name)
-				}
-			}
+			// if l.conkey != "" {
+			// 	idxs := colkeyToInts(l.conkey)
+			// 	for _, idx := range idxs {
+			// 		l.constraint.Columns = append(l.constraint.Columns, table.Columns[idx-1].Name)
+			// 	}
+			// }
 		}
 		if l.referenceTable != "" {
 			referenceTable, err := s.FindTableByName(l.referenceTable)
@@ -343,12 +343,12 @@ ORDER BY x.indexrelid
 				return err
 			}
 			l.constraint.ReferenceTable = &referenceTable.Name
-			if l.confkey != "" {
-				idxs := colkeyToInts(l.confkey)
-				for _, idx := range idxs {
-					l.constraint.ReferenceColumns = append(l.constraint.ReferenceColumns, referenceTable.Columns[idx-1].Name)
-				}
-			}
+			// if l.confkey != "" {
+			// 	idxs := colkeyToInts(l.confkey)
+			// 	for _, idx := range idxs {
+			// 		l.constraint.ReferenceColumns = append(l.constraint.ReferenceColumns, referenceTable.Columns[idx-1].Name)
+			// 	}
+			// }
 		}
 	}
 

--- a/drivers/postgres/postgres.go
+++ b/drivers/postgres/postgres.go
@@ -416,12 +416,11 @@ SELECT
   pc.conname AS name,
   pg_get_constraintdef(pc.oid) AS def,
   contype AS type,
-  psf.relname,
-  pc.conkey::text,
-  pc.confkey::text
+  NULL,
+  NULL,
+  NULL
 FROM pg_constraint AS pc
 LEFT JOIN pg_stat_user_tables AS ps ON ps.relid = pc.conrelid
-LEFT JOIN pg_stat_user_tables AS psf ON psf.relid = pc.confrelid
 WHERE ps.relname = $1
 AND ps.schemaname = $2
 ORDER BY pc.conrelid, pc.conname`

